src/applications/validate-mhv-account/containers/VerifyIdentity.jsx

```diff
@@ -1,28 +1,42 @@
 import React from 'react';
+import { connect } from 'react-redux';
+
 import MessageTemplate from './../components/MessageTemplate';
-import { verify } from '../../../platform/user/authentication/utilities';
 
-const VerifyIdentity = () => {
-  const content = {
-    heading: 'Verify your identity to access health tools',
-    body: (
-      <>
-        <p>
-          We take your privacy seriously, and we’re committed to protecting your
-          information. You’ll need to verify your identity before we can give
-          you access to your personal health information.
-        </p>
-        <button
-          onClick={verify}
-          className="usa-button-primary va-button-primary"
-        >
-          Verify your identity
-        </button>
-      </>
-    ),
-  };
+import { ssoe } from 'platform/user/authentication/selectors';
+import { verify } from 'platform/user/authentication/utilities';
+
+export class VerifyIdentity extends React.Component {
```

Comment:
also no need to convert to a class. could have just pulled the `useSSOe` off of props by doing 
`const VerifyIdentity = ({useSSOe}) => {`
and then
`const authVersion = useSSOe ? 'v1' : 'v0'`